lightwood/helpers/device.py

```diff
@@ -6,14 +7,27 @@
 def get_devices():
     if CONFIG.USE_CUDA and torch.cuda.is_available():
         device_str = "cuda"
+        available_devices = torch.cuda.device_count()
+
+        if available_devices > 1:
+            round_robin = os.environ.get('ROUND_ROBIN_GPU', False)
+            if round_robin in ['1', 'true', 'True', True]:
+                round_robin = True
+            else:
+                round_robin = False
+
+            if round_robin:
+                current_gpu = int(os.environ.get('ROUND_ROBIN_GPU_CURRENT', '0'))
+                device_str = 'cuda:' + str(current_gpu % available_devices)
+                os.envrion['ROUND_ROBIN_GPU_CURRENT'] = (str(current_gpu+1))
     else:
         device_str = "cpu"
+
```

Comment:
missing an `available_devices` default value, probably should go inside this else